src/test/java/roomescape/reservation/service/ReservationServiceTest.java

```diff
@@ -0,0 +1,88 @@
+package roomescape.reservation.service;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.LocalTime;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Stream;
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+import roomescape.common.util.DateTime;
+import roomescape.reservation.domain.Reservation;
+import roomescape.reservation.domain.ReservationRepository;
+import roomescape.reservation.dto.ReservationRequest;
+import roomescape.reservationTime.domain.ReservationTime;
+import roomescape.reservationTime.domain.ReservationTimeRepository;
+import roomescape.theme.domain.Theme;
+import roomescape.theme.domain.ThemeRepository;
+
+class ReservationServiceTest {
+
+    private ReservationService reservationService;
+
+    @BeforeEach
```

Comment:
```suggestion
    private ReservationService reservationService;

    @BeforeEach
```

junit5부터는 기본 라이프사이클이 테스트 메서드당이기 때문에 setUp이 필요없습니다. 

https://junit.org/junit5/docs/current/user-guide/#writing-tests-test-instance-lifecycle

> 함수(또는 메서드)의 길이가 10라인을 넘어가지 않도록 구현한다
beforeEach의 메서드 길이가 필요이상으로 긴 것 같은데 그라운드룰인 10줄 이하로 줄여보세요. 테스트메서드도 똑같은 메서드입니다.

Reply:
헉! 처음안 사실이네요! 좋은 정보 감사합니다